internal/access/access_test.go

```diff
@@ -41,13 +41,8 @@ func setupAccessTestContext(t *testing.T) (*gin.Context, *gorm.DB, *models.Provi
 	c, _ := gin.CreateTestContext(httptest.NewRecorder())
 	c.Set("db", db)
 
-	err := data.SaveSettings(db, &models.Settings{
```

Comment:
My last change (commit 5f6492f0c9bcae58c59ab6917a0768d4371182d1) should fix this, and `credential_test.go` below this.

Reply:
Oh, also, the difference here is on org_collab we are creating a default org for the tests in `setupDB()` in `access.go`. That's covered in commit fe6121273.

Reply:
I saw your change in https://github.com/infrahq/infra/commit/5f6492f0c9bcae58c59ab6917a0768d4371182d1 but I took a different approach here.

I think calling `SaveSettings` in this kind of helper ends up causing more test failures as we make code changes. We should either set this in `setupDB` for everything, or do it in specific tests that require it.  I opted for specific tests.

I think #2902 will remove the need to create a default org in tests. It should exist as soon as we load the database. Although saying it now, I realize there is something missing from that PR to do it as part of initial schema creation.

Reply:
OK. I'm fine with the changes. I think setting the settings in the specific tests ends up being slightly more confusing, because there's really not a case where you wouldn't already have an Org (and consequently settings attached to the org). For example, in `TestSettingsPasswordRequirements` I think getting the current settings and then saving them again is less confusing than pushing in a new set of settings just because that's the way we would expect developers to write their code.

This isn't a huge point though, so let's just move the ball forward.

Reply:
Yes, good point. `SaveSettings` does exactly that now. It updates the settings for the org.

Reply:
Done in #2921, now the only calls to SaveSettings are for testing something that changes settings.